README.md

```diff
@@ -2,3 +2,5 @@
 
 Dieses Repository dient der Ablage der Beispieldateien aus dem Unterricht.
 Wer ist dabei?
+
+Homepage: Flo, Muhammed, Mark
```

Comment:
Das muss ich mir aber noch überlegen :-)

Reply:
Burnout bevor das Praktikum überhaupt beginnt weil unser Vorgesetzter gnadenlos ist :(

Reply:
!!Vorsicht Schadcode

Reply:
bitte nicht in chat ausarten lassen, also bitte....